examples/examples.js

```diff
@@ -191,197 +191,211 @@ async function getTransactionErrorMessageExample(apiClient) {
 }
 
 async function webhookParseExample(apiClient) {
+  const webhookHeader = `{
+    "Authorization-Nonce": "authorization-nonce",
+    "Authorization-Key": "authorization-key",
+    "Authorization-Signature": "authorization-signature"
+  }`;
```

Comment:
similar with the ruby case the headers should be a perfectly normal hash. The user of this will receive this details from the client API, and they all return the headers in some kind of hash or multi-hash